src/unix/solarish/mod.rs

```diff
@@ -1845,6 +1864,13 @@ pub const TIOCMGET: ::c_int = tIOC | 29;
 pub const TIOCREMOTE: ::c_int = tIOC | 30;
 pub const TIOCSIGNAL: ::c_int = tIOC | 31;
 
+pub const DP_POLL: ::c_int = 0xd001;
+pub const DP_ISPOLLED: ::c_int = 0xd002;
+#[cfg(target_os = "illumos")]
+pub const DP_PPOLL: ::c_int = 0xd003;
+#[cfg(target_os = "illumos")]
+pub const DP_EPOLLCOMPAT: ::c_int = 0xd004;
```

Comment:
While this does appear in the devpoll header, it's effectively private for the epoll shim in libc and probably isn't useful to expose for `/dev/poll` consumers in rust.